spec/features/notes_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe "Notes", type: :feature do
+  before(:each) do
+    @alice = User.new(id: 1, email: "alice@example.com", password: "1234567")
+    assert @alice.save
+    @bob = User.new(id: 2, email: "bob@example.com", password: "9876543")
+    assert @bob.save
+    login(@alice)
+  end
+
+  # TODO: Split up
```

Comment:
TODO - how can we split this up?

Reply:
I actually wouldn't split that up since this spec file describes the note controller, and creating a node IS part of the node controller